AutomatedTesting/Gem/PythonTests/EditorPythonBindings/ComponentCommands_test.py

```diff
@@ -24,6 +24,8 @@
 @pytest.mark.parametrize('level', ['Simple'])
 class TestComponentCommands(object):
 
+    # This test was using 'WaterSample' as its test level but the level has been removed.
+    # It needs a new test level in prefab format to make it testable again.
```

Comment:
repeated: Since the 'WaterSample' level has already been deleted from the repo, this comment doesn't add much value. I suggest removing references to the deleted level and just say that it needs a prefab level instead of legacy slice level to work.